liblangtag will fail pkgconf test pipeline

If the pkgconf test pipeline is added to liblangtag it will fail in the following way:

```
2024/11/12 23:12:15 INFO running step "test/pkgconf"
2024/11/12 23:12:15 WARN + '[' -d /home/build ] uses=test/pkgconf
2024/11/12 23:12:15 WARN + cd /home/build uses=test/pkgconf
2024/11/12 23:12:15 WARN + exit 0 uses=test/pkgconf
2024/11/12 23:12:15 INFO running step "pkgconf build dependency check" uses=test/pkgconf
2024/11/12 23:12:16 WARN + '[' -d /home/build ] uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:16 WARN + cd /home/build uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:16 WARN + basename /home/build/melange-out/liblangtag-dev uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:16 WARN + dev_pkg=liblangtag-dev uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:16 WARN + cd / uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:16 WARN + apk info -L liblangtag-dev uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:16 WARN + grep '\.pc$' uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:16 WARN WARNING: opening /home/user/src/wolfi-os/packages: No such file or directory uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:16 WARN WARNING: opening from cache https://packages.wolfi.dev/os: No such file or directory uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:16 WARN + grep -q ^Name: usr/lib/pkgconfig/liblangtag.pc uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:16 WARN + basename usr/lib/pkgconfig/liblangtag.pc .pc uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:16 WARN + lib_name=liblangtag uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:16 WARN + echo usr/lib/pkgconfig/liblangtag.pc uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:16 WARN + grep -q '^usr/lib/pkgconfig/liblangtag.pc$\|^usr/share/pkgconfig/liblangtag.pc$' uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:16 WARN + pkgconf --exists liblangtag uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:16 WARN + pkgconf --print-errors --exists liblangtag uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:16 WARN Package libxml-2.0 was not found in the pkg-config search path. uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:16 WARN Perhaps you should add the directory containing `libxml-2.0.pc' uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:16 WARN to the PKG_CONFIG_PATH environment variable uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:16 WARN Package 'libxml-2.0', required by 'liblangtag', not found uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:16 INFO pod 211f4cc8bb51095ce809552e0631211eb3d6e413e8bbccd31456f21e7833db48 terminated
2024/11/12 23:12:16 INFO ERROR: failed to test package. the test environment has been preserved:
2024/11/12 23:12:16 INFO   workspace dir: /home/user/tmp/melange-workspace-1417894867
2024/11/12 23:12:16 INFO   guest dir: /home/user/tmp/melange-guest-1461723834
2024/11/12 23:12:16 ERRO failed to test package: unable to run pipeline: unable to run pipeline: unable to run pipeline: task exited with code 1
make: *** [Makefile:147: test/liblangtag] Error 1
```

--- a/liblangtag.yaml
+++ b/liblangtag.yaml
@@ -2,7 +2,7 @@
 package:
   name: liblangtag
   version: 0.6.7
-  epoch: 0
+  epoch: 1
   description: Interface library to access/deal with tags for identifying languages
   copyright:
     - license: LGPL-3.0-or-later OR MPL-2.0
@@ -41,8 +41,12 @@ subpackages:
     dependencies:
       runtime:
         - liblangtag
+        - libxml2-dev
         - glib-dev
     description: liblangtag dev
+    test:
+      pipeline:
+        - uses: test/pkgconf
 
   - name: liblangtag-doc
     pipeline:

